resumebuilder/templates/layout.html

```diff
@@ -23,9 +23,12 @@
   <!-- Links -->
 <div id="collapsibleNavbar" class="collapse navbar-collapse">
   <ul class="navbar-nav ml-auto">
-      {%  if current_user.is_authenticated %}
+      {% if current_user.is_authenticated %}
+          <li>
```

Comment:
give <li> tag class of nav-item and <a> tag class of nav-link for consistency and css styling.